docs/transaction.md

```diff
@@ -121,6 +121,11 @@ ARGUMENTS
   ADDRESS  Address of the recipient.
 
 OPTIONS
+  -d, --data=data
+      Specifies the data for the transaction type 0. Takes a string.
```

Comment:
This should be changed based on the flag description

Reply:
Done 👍